alloydb-jdbc-connector/src/main/java/com/google/cloud/alloydb/IpType.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.alloydb;
+
+public enum IpType {
+  PUBLIC,
+  PRIVATE,
+  PSC;
```

Comment:
We are not adding PSC support to the connectors just yet I believe. I think we want to wait on this until we have tested appropriately...

Reply:
updated